src/property.h

```diff
@@ -110,15 +110,17 @@ struct Property : public detail::Property {
     PROPERTY(ABLUFT_IST, 0x0599);
     PROPERTY(FORTLUFT_SOLL, 0x059a);
     PROPERTY(FORTLUFT_IST, 0x059b);
-    PROPERTY(PUMPENZYKLEN_MIN_AUSSENT, 0x05bb);
+    //PROPERTY(PUMPENZYKLEN_MIN_AUSSENT, 0x05bb);
+    PROPERTY(AKTIVE_HEIZSTUFEN, 0x05bb); //THZ404
     PROPERTY(KUEHLSYSTEM, 0x0613);
     PROPERTY(DRUCK_HEIZKREIS, 0x064a, Type::et_dec_val);
     PROPERTY(LEISTUNG_AUSLEGUNG_KUEHLEN, 0x0692);
     PROPERTY(MOTORSTROM, 0x069f);
     PROPERTY(MOTORLEISTUNG, 0x06a0, Type::et_cent_val);
     PROPERTY(MOTORSPANNUNG, 0x06a1);
     PROPERTY(ABLUFTTEMP, 0x0694, Type::et_dec_val);
-    PROPERTY(ANZEIGE_NIEDERDRUCK, 0x07a7, Type::et_dec_val);
+    PROPERTY(ANZEIGE_HOCHDRUCK, 0x07a6, Type::et_cent_val); //NEU
```

Comment:
If you add new values to property.h please guard it with the corresponding flag for your heatpump, no additional comments are needed
```
#ifdef THZ404
PROPERTY(ANZEIGE_HOCHDRUCK, 0x07a6, Type::et_cent_val);
#endif
```
others with different heatpumps can then try it int heir setup and add their flag to enable it ... once all variants confirmed that it works we can set it as default and remove the flags

Reply:
Great, the example helps me and I'll test it later :-)